spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -250,39 +250,39 @@ destination of messages it is sending and the source of messages it
 receives.
 
 Administered objects are placed in a JNDI namespace by an administrator.
-A JMS client typically notes in its documentation the JMS administered
+A Jakarta Messaging client typically notes in its documentation the Jakarta Messaging administered
 objects it requires and how the JNDI names of these objects should be
-provided to it. Figure 2‑1 illustrates how JMS administration ordinarily
+provided to it. Figure 2‑1 illustrates how Jakarta Messaging administration ordinarily
 works.
 
-Figure 2‑1 JMS Administration
+Figure 2‑1 Jakarta Messaging Administration
 
 image:1.png[image,width=390,height=213]
 
 === Two messaging styles
 
-A JMS application can use either the point-to-point (PTP) or the
+A Jakarta Messaging application can use either the point-to-point (PTP) or the
 publish-and-subscribe (pub/sub) style of messaging, which are described
 in more detail later in this specification. An application can also
 combine both styles of messaging in one application. These two styles of
-messaging are often referred to as messaging domains. JMS provides these
+messaging are often referred to as messaging domains. Jakarta Messaging provides these
 two messaging domains because they represent two common models for
 messaging.
 
-When using the JMS API, a developer can use interfaces and methods that
+When using the Jakarta Messaging API, a developer can use interfaces and methods that
 support both models of messaging. When using these interfaces, the
 behavior of the messaging system may be somewhat different, because the
 two messaging domains have different semantics. These semantic
 differences are described in chapter 4 “Messaging domains”.
 
-=== JMS APIs
+=== Jakarta Messaging APIs
 
-For historical reasons JMS offers four alternative sets of interfaces
+For historical reasons Jakarta Messaging offers three alternative sets of interfaces
```

Comment:
Why change to three - (1.0 = 2, 1.1 = 1, 2.0 = 1 = four)

Reply:
I reverted this back to "four"